docker/src/main/java/brooklyn/entity/container/docker/DockerHostImpl.java

```diff
@@ -869,6 +876,29 @@ public boolean apply(String input) {
         }
     }
 
+    @Override
+    protected void connectServiceUpIsRunning() {
+        HostAndPort brooklynAccessibleAddress = BrooklynAccessUtils.getBrooklynAccessibleAddress(this, this.getAttribute(this.DOCKER_SSL_PORT));
+        String url = String.format("https://%s:%d/info", brooklynAccessibleAddress.getHostText(), brooklynAccessibleAddress.getPort());
+
+        serviceUpIsRunningFeed = SshFeed.builder()
```

Comment:
**TODO** change to `HttpFeed` with client certificates at some point in the future...